lib/pbench/test/unit/server/query_apis/test_table_of_content.py

```diff
@@ -407,6 +407,6 @@ def test_empty_query(
             assert expected_result == res_json
 
     def test_get_index(self, attach_dataset, provide_metadata):
-        drb = Dataset.attach(controller="node", name="drb")
+        drb = Dataset.attach(name="drb")
```

Comment:
This is fine, but as a "usage note" ... `Dataset.attach` is a wrapper around `Dataset.query`, and the only real advantage of `Dataset.attach` is that it will analyze a file `path` parameter to find the dataset name, and it allows you to change the state of the dataset at the same time. If you're not doing either of those things (and you're not, here), using `Dataset.query(name="drb")` would generally be preferable in the future.